libminifi/src/utils/RegexUtils.cpp

```diff
@@ -22,58 +22,86 @@
 #include <vector>
 
 #include "Exception.h"
+#include <regex.h>
```

Comment:
Is this `#include` necessary?  `regex.h` is already included in the header in the non-standard-lib case; also both MSVC and the linter dislike it.

Reply:
removed